src/Aspire.Dashboard/Components/Pages/ConsoleLogs.razor

```diff
@@ -27,24 +27,30 @@
                             @bind-SelectedResource:after="HandleSelectedOptionChangedAsync"
                             LabelClass="toolbar-left" />
 
+            <PauseIncomingDataSwitch IsPaused="@PauseManager.ConsoleLogsPaused" IsPausedChanged="@IsPausedChangedAsync" />
             <ClearSignalsButton SelectedResource="PageViewModel.SelectedOption" HandleClearSignal="ClearConsoleLogs" />
 
-            @foreach (var command in _highlightedCommands)
+            @if (_highlightedCommands.Count > 0)
```

Comment:
Nothing in this block actually changed other than adding a Commands label when there are highlighted commands to separate from the built-in signals/pause commands